pallets/ddc-staking/src/lib.rs

```diff
@@ -737,9 +835,94 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		/// Pay out all the stakers for a single era.
+		#[pallet::weight(100_000)]
+		pub fn payout_stakers(origin: OriginFor<T>, era: EraIndex) -> DispatchResult {
+			ensure_signed(origin)?;
+			let current_era = Self::current_era().ok_or(Error::<T>::DDCEraNotSet)?;
+
+			// Makes sure this era hasn't been paid out yet
+			ensure!(!Self::paid_eras(era), Error::<T>::DoubleSpendRewards);
+
+			// This should be adjusted based on the finality of validation
+			ensure!(current_era >= era + 2, Error::<T>::EraNotValidated);
```

Comment:
How does it depend on the finality? Why `+2`?

Reply:
suggestions are welcomed